### PR TITLE
Basic features

### LINKED ISSUE #1: API POST Handler
The collector should receive POST request with the following format and store them in a DB: 
```json
{
   "serverName":"my-Server-01",
   "vulnerableFiles":[
      "c:\temp\funnyClass.jar",
      "d:\\dev\\Reader.jar"
   ],
  "reportTime": $CurrentDateTime (automatically generated)
}
```
If a record with the same serverName exists, overwrite "vulnerableFiles" and "reportTime" with the new values. We do not track history. If the new POST to the existing serverName has an empty "vulnerableFiles" array, remove the item from the DB as it nolonger has vulnerable files.

With GET you should get the same JSON format as above back.

### PR DESCRIPTION
**Pull request by janlauber**
Facing the following issue(s):
- #1

### CODE REVIEW

**Review by michaelmumenthaler — CHANGES_REQUESTED** [review]
remove funding

File: FUNDING.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# These are supported funding model platforms
+
+github: [janlauber]
```
Comment (michaelmumenthaler) [inline FUNDING.yml]: Remove funding

Comment (janlauber) [reply FUNDING.yml]: done

**Review by michaelmumenthaler — APPROVED** [review]
LGTM

**Comment by michaelmumenthaler** [comment]
LGTM

**Pull request merged**